test/quantization/test_workflow_module.py

```diff
@@ -670,3 +677,99 @@ def test_fq_serializable(self):
         loaded_dict = torch.load(b)
         for key in state_dict:
             self.assertEqual(state_dict[key], loaded_dict[key])
+
+def _get_buffer_ids(module):
+    """
+    Object addresses stay constant if and only if all modifications are in-place
+    """
+    return [id(v) for k, v in module._buffers.items()]
+
+class TestDistributed(QuantizationTestCase):
```

Comment:
This is awesome, I think we should also add a test for DDP here at some point.

Reply:
I agree fully. For the tests below, is there any value in making sure that they work with actual calls to dataparallel? The main value I can think of is if someone makes changes to dataparallel, we can catch that if our tests fail.

Reply:
I added comments in the code and commented regarding this on your comment below, hope that's ok.  I'm flexible.